tests/e2e/lib.sh

```diff
@@ -331,6 +335,40 @@ setup_podsecuritypolicies_config() {
     fi
 }
 
+# wait_for_collectors_to_be_operational() ensures that collector pods are able
+# to load kernel objects and create network connections.
+wait_for_collectors_to_be_operational() {
+    info "Will wait for collectors to reach a ready state"
+
+    local readiness_indicator="Successfully established GRPC stream for signals"
```

Comment:
What if the log statement in collector changes? I think it would be better to check something like if collector reports its own process or something like that.

Reply:
I agree this is a bit fragile (but if someone updates the collector they will have to fix this). I'm happy to check something else if you have a concrete suggestion.

Reply:
I suggested checking if collector has reported its own process above, but that might not be the best way as that would rely on sensor and central. You could check collector metrics and make sure that the number of processes or network connections is not 0.